sync/private_test_summarizer.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+
+class PrivateTestSummarizerException(Exception):
```

Comment:
The name of the exception should indicate when it was raised. Maybe a better name might be `SummaryWriteError`?